Source/VCGeneration/VerificationConditionGenerator.cs

```diff
@@ -1365,23 +1351,22 @@ public static Counterexample AssertCmdToCounterexample(VCGenOptions options, Ass
     /// <summary>
     /// Returns a clone of "cex", but with the location stored in "cex" replaced by those from "assrt".
     /// </summary>
-    public static Counterexample AssertCmdToCloneCounterexample(VCGenOptions options, AssertCmd assert,
+    public static Counterexample AssertCmdToCloneCounterexample(VCGenOptions options, AssertCmd assrt,
       Counterexample cex,
-      Block implEntryBlock, Dictionary<TransferCmd, ReturnCmd> gotoCmdOrigins)
+      Block implEntryBlock)
     {
-      Contract.Requires(assert != null);
+      Contract.Requires(assrt != null);
```

Comment:
Strrrrrange renaming...

Reply:
A change from the refactoring PR you reviewed before, that through a git error of mine did not end up in this PR.